templates/user/settings.tmpl

```diff
@@ -12,6 +12,12 @@ h3 { font-weight: normal; }
 		{{range .Flashes}}<li class="urgent">{{.}}</li>{{end}}
 	</ul>{{end}}
 
+	{{if .Suspended}}
+	<div class="alert info">
+		<p>This account is currently suspended.</p>
+		<p>Please contact the instance administrator to discuss reactivation.</p>
+	</div>
+	{{end}}
```

Comment:
I think we should get this into a user template so we can also display it on all user backend pages (e.g. /me/c/, /me/posts/, etc.)